components/caching-invalidator/org.wso2.carbon.caching.invalidator/pom.xml

```diff
@@ -48,7 +48,7 @@
         <repository>
             <id>wso2-nexus</id>
             <name>WSO2 internal Repository</name>
-            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <url>https://maven.wso2.org/nexus/content/groups/wso2-public/</url>
```

Comment:
Thanks for your PR, however, We usually don't recommend having <repositories> declarations in component level. Hence can you please remove the <repositories> section and resend the PR.

Reply:
I have 'Allow edits from maintainers.' checked. Please feel free to make the configuration change to this PR.